utils/markdown/renderer.tsx

```diff
@@ -34,7 +34,9 @@ export default class Renderer extends marked.Renderer {
         if (usedLanguage === 'tex' || usedLanguage === 'latex') {
             return `<div data-latex="${TextFormatting.escapeHtml(code)}"></div>`;
         }
-
+        if (usedLanguage === 'texcode' || usedLanguage === 'latexcode') {
+            usedLanguage = 'tex';
+        }
         // treat html as xml to prevent injection attacks
```

Comment:
Could you please add a empty line here to fix the linter issue? https://app.circleci.com/pipelines/github/mattermost/mattermost-webapp/21393/workflows/cde51fde-321d-41d7-b8dd-402fc7abcda8/jobs/135465/parallel-runs/0/steps/0-103

Reply:
Done and pushed @hanzei. :)

Reply:
The linter now complains about a white space:
 ```
/root/mattermost/mattermost-webapp/utils/markdown/renderer.tsx
 
  40:1  error  Trailing spaces not allowed  no-trailing-spaces
 ```

Reply:
Checking @hanzei

Reply:
@hanzei Pushed the required changes. :)